bin/check-reserved-instances.rb

```diff
@@ -0,0 +1,107 @@
+#! /usr/bin/env ruby
+#
+# check-reserved-instances
+#
+# DESCRIPTION:
+#   This plugin checks if reserved instances expire soon.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#
+# USAGE:
+#   ./check-reserved-instances.rb --aws-region eu-west-1 --use-iam
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright (c) 2015, Olivier Bazoud, olivier.bazoud@gmail.com
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
```

Comment:
include Common and follow the pattern of 

https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/bin/check-ebs-snapshots.rb